oca_dependencies.txt

```diff
@@ -1 +1,2 @@
 # See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
+odoo14-addon-account-payment-order @ git+https://github.com/OCA/bank-payment.git@refs/pull/451/head#subdirectory=setup/account_payment_order
```

Comment:
@hkapatel-initos should'nt the link reference the account-payment repo instead of bank-payment? I think you've got the repositories mixed up.